pypeman/channels.py

```diff
@@ -24,6 +24,17 @@
 MSG_CTXVAR = contextvars.ContextVar("msg")
 
 
+def get_channel(name):
+    """Permits retrieving a channel by its name or short_name
+
+    Args:
+        name (str): The channel's name
+    """
+    for chan in all_channels:
+        if chan.name == name or chan.short_name == name:
```

Comment:
can probably close.

could it ever be that a shortname matches one channel and a name matches another one?

If

Reply:
Nope, chan.name is the complete path of the channel: `parentchan1.parentchan2.short_name` and short_name are unique so i can't find a world that will cause a problem